submissions/VikTorNik/html-css-popup/index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hooli-style Popup</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <div class="wrapper">
+        <header class="header">
+            <nav class="nav">
+                <ul class="nav__menu">
+                    <li><a href="#">File</a></li>
```

Comment:
Please fix the hitbox of menu items. [Watch video](https://share.vidyard.com/watch/ev4eaqs5C5nnyLUFuCSrBJ?)